src/NzbDrone.Core/Download/Clients/DownloadStation/TorrentDownloadStation.cs

```diff
@@ -199,6 +200,11 @@ protected bool IsFinished(DownloadStationTask torrent)
             return torrent.Status == DownloadStationTaskStatus.Finished;
         }
 
+        protected bool IsCompleted(DownloadStationTask torrent)
+        {
+            return torrent.Status == DownloadStationTaskStatus.Seeding || IsFinished(torrent);
```

Comment:
Dejavu. We already had this conversation. I forgot to check on waiting if torrent.Size == 0 || GetRemainingSize(torrent) > 0 ? DownloadItemStatus.Queued : DownloadItemStatus.Completed;